modules/hcp-configure-ntp.adoc

```diff
@@ -0,0 +1,183 @@
+// Module included in the following assemblies:
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-configure-ntp_{context}"]
+= Configuring the NTP server for hosted clusters
+
+You can configure the Network Time Protocol (NTP) server for your hosted clusters by using Butane.
+
+.Procedure
+
+. Create a Butane config file, `99-worker-chrony.bu`, that includes the contents of the `chrony.conf` file.
++
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
+====
++
+.Example `99-worker-chrony.bu` configuration
+[source,yaml,subs="attributes+"]
+----
+variant: openshift
+version: {product-version}.0
+metadata:
+  name: 99-worker-chrony
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+storage:
+  files:
+  - path: /etc/chrony.conf
+    mode: 0644 <1>
+    overwrite: true
+    contents:
+      inline: |
+        pool 0.rhel.pool.ntp.org iburst <2>
+        driftfile /var/lib/chrony/drift
+        makestep 1.0 3
+        rtcsync
+        logdir /var/log/chrony
+----
+<1> Specify an octal value mode for the `mode` field in the machine config file. After creating the file and applying the changes, the `mode` field is converted to a decimal value. You can check the updated field in the YAML file with the `oc get mc <machineconfig_name> -o yaml` command.
+<2> Specify any valid, reachable time source, such as the one provided by your DHCP server.
++
+[NOTE]
+====
+For all-machine to all-machine communication, the NTP on UDP port is `123`. If you configured an external NTP time server, you must open UDP port `123`.
+====
+
+. Use Butane to generate a `MachineConfig` object file, `99-worker-chrony.yaml`, that contains a configuration to be sent to the nodes. Run the following command:
++
+[source,terminal]
+----
+$ butane 99-worker-chrony.bu -o 99-worker-chrony.yaml
+----
+
+. Add the contents of the `99-worker-chrony.yaml` file inside of a config map in the management cluster:
++
+.Example config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <configmap_name>
+  namespace: clusters
+data:
+  config: |
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: worker
+      name: <machineconfig_name>
+    spec:
+      config:
+        ignition:
+          version: 3.2.0
+        storage:
+          files:
+          - contents:
+              source: data:...
+            mode: 420
+            overwrite: true
+            path: <path> <1>
```

Comment:
I think this is already generated and the user needs to literally copy the Machine Config from `99-worker-chrony.yaml` file into this configmap